T3/Gui/OutputUi/FloatDictOutputUi.cs

```diff
@@ -0,0 +1,162 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Numerics;
+using System.Linq;
+using ImGuiNET;
+using T3.Core.Operator.Slots;
+using T3.Core.DataTypes;
+using T3.Gui;
+using Vector2 = System.Numerics.Vector2;
+
+namespace T3.Gui.OutputUi
+{
+    public class FloatDictOutputUi : OutputUi<Dict<float>>
+    {
+        public override IOutputUi Clone()
+        {
+            return new FloatDictOutputUi()
+            {
+                OutputDefinition = OutputDefinition,
+                PosOnCanvas = PosOnCanvas,
+                Size = Size
+            };
+        }
+
+        private static bool _autoFit = true;
+        private static bool _clear = false;
+        private static float _minFit = -1;
+        private static float _maxFit = 1;
+
+        private static List<List<float>> _previousValues;
+        
+        protected override void DrawTypedValue(ISlot slot)
+        {
+            if (slot is Slot<Dict<float>> typedSlot)
+            {
+                ImGui.BeginChild("##scrolling");
+
+                var v = typedSlot.Value;
+                var outputString = v == null ? "" : string.Join(", ", $"{v:0.000}");
+                ImGui.TextUnformatted($"{outputString}");
+
+                if (v == null)
+                    return;
+
+                var count = Math.Min(1024, v.Count);
+                var horizontalLength = 512;
+
+                if (_clear)
+                    _previousValues = null;
+
+                // expand list of previous values if necessary
+                if (_previousValues == null)
+                    _previousValues = new List<List<float>>();
+                if (count > _previousValues.Count)
+                {
+                    while (_previousValues.Count < count)
+                    {
+                        // and new list and initialize to zeroes
+                        var newList = Enumerable.Repeat(0f, horizontalLength).ToList();
+                        _previousValues.Add(newList);
+                    }
+                }
+
+                var valueEnum = v.Values.GetEnumerator();
```

Comment:
```suggestion
                var enumarator = v.Values.GetEnumerator();
```

But why using enumerator instead of `foreach(` ?

Reply:
...lack of C# experience from my side again.